test-harness/specs/manually-select-examples-AC-0.oas2.txt

```diff
@@ -0,0 +1,23 @@
+====test====
+Trying to access a default example when no example is defined, fails with 404.
+====spec====
```

Comment:
If we’re removing default let’s remove default? Also let’s not have a test which requests default?

Reply:
Right. This is a leftover, removed in 238e060